linear_operator/operators/dense_linear_operator.py

```diff
@@ -76,13 +76,7 @@ def __add__(self, other):
         elif isinstance(other, torch.Tensor):
             return DenseLinearOperator(self.tensor + other)
         else:
-            return super(DenseLinearOperator, self).__add__(other)
-
-    def mul(self, other):
```

Comment:
As long as the case for `DenseLinearOperator` exists in `_mul_matrix` in `_linear_operator.py`, this is not necessary.